tests/test_oauthmanager.py

```diff
@@ -76,27 +76,50 @@ def save_oidc_token(account, lifetime_access=0, lifetime_refresh=0, refresh_toke
 
 def get_oauth_session_param_count(account):
     session = get_session()
-    result = session.query(models.OAuthRequest).filter_by(account=account).all()  # pylint: disable=no-member
-    return len(result)
+    stmt = select(
+        func.count()
+    ).select_from(
+        models.OAuthRequest
+    ).where(
+        models.OAuthRequest.account == account
+    )
+    return session.execute(stmt).scalar()
 
 
 def get_token_count(account):
     session = get_session()
-    result = session.query(models.Token).filter_by(account=account).all()  # pylint: disable=no-member
+    stmt = select(
+        models.Token
+    ).where(
+        models.Token.account == account
+    )
+    result = session.execute(stmt).scalars().all()
     for token in result:
         print(token.token, token.expired_at, token.refresh_token, token.refresh_expired_at, token.oidc_scope)
     return len(result)
```

Comment:
If we replace this with a `func.count()` then we wouldn't be able to index the columns for the print statement (which may or may not be important, I don't know). It is possible to do something like:

```
def get_token_count(account):
    session = get_session()
    stmt = select(
        models.Token,
        func.count().over().label('token_count')
    ).select_from(
        models.Token
    ).where(
        models.Token.account == account
    )
    result = session.execute(stmt).all()
    for token, count in result:
        print(token.token, token.expired_at, token.refresh_token, token.refresh_expired_at, token.oidc_scope)
    return count
```

This also carries some overhead as the column 'label_count' is included in every row. It is also possible to do two queries, one for the count and one for the rows, this will have some overhead too. Maybe the best would be to remove the prints? It does not show in the test output nor in `logshow`

Reply:
I'd be in favour of removing the print statement here (related: https://github.com/rucio/rucio/issues/6718)

Reply:
Removed the prints and refactored this to use func.count() instead in a separate commit - 00cffa126f49b448df14231361e4b7b4a36fa979